acceptance/tests/fixtures/cases/terminating-gateway-namespaces/all-non-default/external-service-registration/external-service.yaml

```diff
@@ -5,7 +5,6 @@ apiVersion: consul.hashicorp.com/v1alpha1
 kind: Registration
 metadata:
   name: static-server-registration
```

Comment:
Is this just because we don't care where the registration actually lives on the cluster?

Reply:
yeah, you can have a service in your k8s cluster that is no part of your mesh and have a terminating gateway front it which is what we're doing here